tests/spec_decode/e2e/conftest.py

```diff
@@ -263,6 +285,7 @@ def run_greedy_equality_correctness_test(baseline_llm_generator,
         max_tokens=max_output_len,
         ignore_eos=ignore_eos,
         temperature=temperature,
+        seed=seed,
```

Comment:
nit: this should change seed per request

Reply:
done